corral/cmd/context.pony

```diff
@@ -9,16 +9,19 @@ class val Context
   let log: Log
   let quiet: Bool
   let nothing: Bool
+  let directory: String
```

Comment:
If you are willing to try something out, you might be able to fix another issue while you're at it. If this directory were a FilePath, we could construct it up here with env.root, and wouldn't need to access env.root down in the utility functions. I'm not sure if that will be simple or create more work.

Reply:
Hmm, we would still need to build another `FilePath` while searching for the `corral.json` file.

Reply:
OK, no worries. I'll try that out later.